ecs-init/apparmor/apparmor.go

```diff
@@ -28,11 +30,22 @@ const (
 	appArmorProfileDir         = "/etc/apparmor.d"
 )
 
+type profile struct {
+	IncludeTunablesGlobal   bool
+	IncludeAbstractionsBase bool
+}
+
 const ecsAgentDefaultProfile = `
+{{if .IncludeTunablesGlobal}}
 #include <tunables/global>
+{{else}}
+@{PROC}=/proc/
+{{end}}
 
```

Comment:
Do we know why we include `tunables/global` and what does `@PROC=/proc/` do?

Reply:
`tunables/global` contains global variable definitions that can be used in AppArmor profiles. One of those variables is `@{PROC}`, which is where `procfs` is mounted. We need `@{PROC}` defined because we use it in the agent profile.

Contents of `tunables/global`:
```
# All the tunables definitions that should be available to every profile
# should be included here

include <tunables/home>
include <tunables/multiarch>
include <tunables/proc>
include <tunables/alias>
include <tunables/kernelvars>
include <tunables/xdg-user-dirs>
include <tunables/share>
include <tunables/etc>
include <tunables/run>

# Also, include files in tunables/global.d for site-specific adjustments
include if exists <tunables/global.d>
```

Contents of `tunables/proc`:
```
# @{PROC} is the location where procfs is mounted.
@{PROC}=/proc/

# Also, include files in tunables/proc.d for site-specific adjustments
include if exists <tunables/proc.d>
```

Reply:
Thanks for explaining!